roles/KubevirtCommonTemplatesBundle/filter_plugins/k8s_inject_ownership.py

```diff
@@ -0,0 +1,26 @@
+from ansible.errors import AnsibleError
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'k8s_inject_ownership': k8s_inject_ownership
+        }
+
+def k8s_inject_ownership(objects, owner):
+    owner_ref = {
+        "apiVersion": owner["apiVersion"],
+        "kind": owner["kind"],
+        "name": owner['metadata']["name"],
+        "uid": owner['metadata']['uid']
+    }
+
+    ownerUID = owner['metadata']['uid']
+
+    for obj in objects:
```

Comment:
This part seems a bit odd to me, I don't see how it adds `owner_ref` to a template (`object`) that doesn't already have it